python/GafferDispatchUI/PythonCommandUI.py

```diff
@@ -105,9 +112,17 @@
 			# Do some one-time finalization
 			...
 			```
+
+			> Note : In Single mode, the command will only be called for each
+			> frame if the inputs are animated. If the inputs are static
+			> then the command will only be called once.
```

Comment:
I wonder if there's a future where the current Single mode becomes "Default" or "Automatic" and we introduce a new "Frame" mode (name pending...) that satisfies the `PythonCommand2` creators of this world?

Reply:
I _think_ that would be an orthogonal feature. The "always run" thing applies to any changes in context, not just the frame. In the case of `Dispatcher::hash()` I've been thinking of a single `deduplicate` plug, that when off, appends the whole context hash. Does that make sense?

Reply:
> I've been thinking of a single deduplicate plug, that when off, appends the whole context hash. Does that make sense?

Cool, makes sense. That'd also give us a reasonable place to document the behaviour.